### ISSUE TITLE
Glob operator !! does not return a seq<string>

### ISSUE THREAD

**Issue by gkalnytskyi** [fsharp/FAKE#2062]
### Description

I encountered and issue when glob operator `!!` returns `IGlobbingPattern` which is not converted to `seq<String>`. I stumbled upon this issue while trying to run _Compiling the application_ example from [Getting Started](https://fake.build/fake-gettingstarted.html) page using upgraded to .Net 4.6.1 _Fake-Calculator_ solution.

### Repro steps
Here is a minimal example. Assuming you have a folder `TryFake`. it contains subfolder `files` which itself contains the following files:
- a.txt
- aa.txt
- aab.txt
- ab.txt
- abb.txt
- b.txt
- ba.txt
- baa.txt

The Fake script is placed in the `TryFake` folder and contains the following code:

    #r "paket:
    nuget Fake.Core.Target
    nuget Fake.IO.FileSystem //"
    #load "./.fake/tryscript.fsx/intellisense.fsx"

    open Fake.IO
    open Fake.IO.Globbing
    open Fake.IO.Globbing.Operators
    let files = !! "files/a*.txt"

    files
        |> Seq.iter (fun x -> printfn "File: %s" x)

After you run the script you receive the following error: `The type 'IGlobbingPattern' is not compatible with the type 'seq<'a>'`

If you modify a script to

    #r "paket:
    nuget Fake.Core.Target
    nuget Fake.IO.FileSystem //"
    #load "./.fake/tryscript.fsx/intellisense.fsx"
    
    open Fake.IO
    open Fake.IO.Globbing
    open Fake.IO.Globbing.Operators
    let files = !! "files/a*.txt"
    
    for file in files do
        printfn "File: %s" file

it provides you with the correct result of files that start with the letter _a_.

### Expected behavior

Script outputs file names.

### Actual behavior

The following error occurs: `The type 'IGlobbingPattern' is not compatible with the type 'seq<'a>'`

### Related information 

* Operating system: Windows 10 (17134)
* Branch: Fake release 5.3.1 installed using `dotnet tool install fake-cli -g`
* .NET Runtime, CoreCLR or Mono Version: .NET Framework 4.7.2, .Net Core SDK 2.1.302
* Version of FAKE 5.3.1 installed using `dotnet tool install fake-cli -g`

**Comment by gkalnytskyi** [comment]
Hi @matthid,
I have created [this](https://github.com/gkalnytskyi/TryFSharpMake) repository that includes all the files that Fake generated

**Comment by csmager** [comment]
I get this error if I run your script on Windows. If you remove the `.fake` directory and run again, it works.

**Comment by gkalnytskyi** [comment]
Hello @csmager,
Unfortunately, removing `.fake` folder doesn't work for me. After deleting it and rerunning `fake run .\trysript.fsx` I still get the same error.

**Comment by csmager** [comment]
Only difference I can see is the assemblies that are referenced, particularly within the Netstandard.Library 2.0.2 package. You seem to be running the same `dotnet` and `fake` versions as me otherwise.

Perhaps try:
- delete `.fake` directory
- delete NuGet caches `C:\Users\<username>\.nuget`, `%LOCALAPPDATA%\NuGet`

**Comment by gkalnytskyi** [comment]
@csmager , I have deleted all the Nuget caches you specified and `.fake` folder. And then rerun the the `fake` command with and without the lock file and the result was the same. Right after deleting caches I got the following error: `There was a problem while setting up the environment:
-> IOException: The process cannot access the file '<User Dir>\.nuget\packages\netstandard.library\2.0.2\build\netstandard2.0\ref\System.AppContext.dll' because it is being used by another process.
Hint: If you just upgraded the fake-runner you can try to remove the .fake directory and try again.`

After I removed `.fake` folder and retry the old error about `IGlobbingPatter` reappears.

**Comment by matthid** [comment]
We can also compare our verbose output, which might help to identity the issue. I hope I can reproduce your issue but I'm not quite there yet.

**Comment by matthid** [comment]
@gkalnytskyi Can you clean everything and send the output of `fake -vv run tryscript.fsx` and attach the logfile? Something is indeed fishy with the NetStandard.Library package

Can you also compress and attach the folder `C:\Users\<username>.nuget\packages\netstandard.library` and then try to delete it (and again create a logfile for that)?

**Comment by gkalnytskyi** [comment]
Hi @matthid, after a Windows 10 update and clearing up all the caches again I can no longer reproduce the issue. I could reproduce it right after Windows 10 was updated, but before I cleared caches, but I didn't create the extended logs, or saved folder contents at that time.

I think this issue can be closed. Sorry for the hassle. If I encounter similar issue again, I'll create better logs and save the contents of the `netstandard.library` folder.
